website/src/css/customTheme.css

```diff
@@ -707,7 +707,7 @@ a:hover {
 .plugin-logo {
   margin-top: 10px;
   max-width: 250px;
-  max-height: 180px;
+  height: 180px;
```

Comment:
I noticed that after this change, the icon in plugins page will get larger than before. In fact we have already limit the `max-width` here, we can we can let the height zone adjust itself in equal proportions, meanwhile smaller will make the page look more sophisticated, what do you think ?
![image](https://user-images.githubusercontent.com/2561857/130519119-96e48197-e970-4659-8fb2-a0c38c2ec0dd.png)



cc @1502shivam-singh @foolwc @bzp2010 @Baoyuantop @KishaniKandasamy

Reply:
I changed it because the contents in plugin cards cannot align to the same horizontal line.  I will fix it later :)
  
<img width="925" alt="截屏2021-08-24 上午7 32 39" src="https://user-images.githubusercontent.com/13642747/130532670-faa6e11a-dd5f-4660-8fce-be7e7ad05bc0.png">

Reply:
> I changed it because the contents in plugin cards cannot align to the same horizontal line.

Something I also wanted to mention earlier, nice of you to do it here. 
You might want to reduce the max-height of the SVGs to get that white space around for some breathing space

Reply:
> I changed it because the contents in plugin cards cannot align to the same horizontal line. I will fix it later :)
> 
> <img alt="截屏2021-08-24 上午7 32 39" width="925" src="https://user-images.githubusercontent.com/13642747/130532670-faa6e11a-dd5f-4660-8fce-be7e7ad05bc0.png">

this can also be fixed in the #513